dev-db/mariadb/mariadb-10.6.11-r1.ebuild

```diff
@@ -0,0 +1,1321 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+SUBSLOT="18"
+
+JAVA_PKG_OPT_USE="jdbc"
+
+inherit systemd flag-o-matic prefix toolchain-funcs \
+	multiprocessing java-pkg-opt-2 cmake
+
+# Patch version
+PATCH_SET="https://github.com/hydrapolic/gentoo-dist/raw/master/mariadb/mariadb-10.6.10-patches-01.tar.xz"
+
+SRC_URI="mirror://mariadb/${PN}-${PV}/source/${P}.tar.gz
+	${PATCH_SET[@]}"
```

Comment:
SRC_URI="
    mirror://mariadb/${PN}-${PV}/source/${P}.tar.gz
    https://github.com/hydrapolic/gentoo-dist/raw/master/${PN}/${P}-patches-01.tar.xz
"

Reply:
Pushed an update, thanks for the review @scardracs

Reply:
You’re welcome 😸